hazelcast/src/main/java/com/hazelcast/nio/Packet.java

```diff
@@ -88,12 +88,35 @@ public void setConn(Connection conn) {
         this.conn = conn;
     }
 
-    public void setFlag(int bit) {
-        flags |= 1 << bit;
+    /**
+     * Sets a particular flag.
```

Comment:
Imho that is implied by the fact that flags are being used. It would be illogical that other flags would be changed.